app/js/controllers/foyer/ressources/enfants.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+angular.module('ddsApp').controller('FoyerRessourcesEnfantsCtrl', function($scope, SituationService, RessourceService) {
+    var momentDebutAnnee = moment($scope.situation.dateDeValeur).subtract('years', 1);
+    $scope.debutAnneeGlissante = momentDebutAnnee.format('MMMM YYYY');
+
+    $scope.enfants = SituationService.getEnfants($scope.situation);
+    $scope.enfants.forEach(function(enfant) {
+    	enfant.hasRessources = RessourceService.getMainScreenRessources(enfant).length > 0 || false;
```

Comment:
Setting the default value of a Boolean to `false` seems pretty weird, as the only way the `false` could be assigned would be if the leftmost part of the expression was itself `false`…